cumulus_library_data_metrics/q_date_in_lifetime/q_date_in_lifetime.jinja

```diff
@@ -0,0 +1,27 @@
+{% import 'utils.jinja' as utils %}
+
+CREATE TABLE {{ study_prefix }}__q_date_in_lifetime_{{ src|lower }} AS
+WITH invalid_fields AS (
+{% for field in dates %}
+    SELECT
+        {{ src }}.{{ patient_field }}.reference as patient_ref,
+        P.date_of_birth AS date_of_birth,
+        P.date_of_death AS date_of_death,
+        id as resource_id,
+        '{{ field }}' AS field,
+        {{ field }}  AS value
+    FROM {{ src }}
+        INNER JOIN PATIENT_LIFESPAN P ON P.ref = {{ src }}.{{ patient_field }}.reference
```

Comment:
extremely minor nit: we don't generally capitalize table names or aliases, just keywords. It doesn't matter, just a tiny style difference.